app/src/util/tools.ts

```diff
@@ -354,6 +354,52 @@ export const formatNumber = (number: string, decimals = 2): string => {
   return `${formattedSubstring}${decimals > 0 ? '.' + fixedInt.split('.')[1] : ''}`
 }
 
+export const formatHistoryDate = (dateData: string): string => {
+  const date = new Date(dateData)
+  const minute = date.getMinutes()
+  const minuteWithZero = (minute < 10 ? '0' : '') + minute
+  const hour = date.getHours()
+  const hourWithZero = (hour < 10 ? '0' : '') + hour
+  return `${date.getDate()}.${date.getMonth()} - ${hourWithZero}:${minuteWithZero}`
```

Comment:
Why is the month value 0 for me?
![image](https://user-images.githubusercontent.com/4677914/104460632-ecdeed00-55e0-11eb-8a7f-6a211f1c0b2b.png)
Maybe the month value needs to be offset by 1?

Reply:
Ohh yeah good catch haven't noticed that will fix it!